systemd/crc-cluster-status.service

```diff
@@ -4,10 +4,13 @@ After=crc-wait-apiserver-up.service crc-pullsecret.service
 After=ocp-mco-sshkey.service ocp-cluster-ca.service
 After=ocp-custom-domain.service ocp-userpasswords.service
 After=ocp-clusterid.service
+StartLimitIntervalSec=450
+StartLimitBurst=10
```

Comment:
> values such that the service  never hits the re-start limit and is always re-started

@anjannath Is there any source/doc/link which states that with using above limits service never hits re-start limit?

Reply:
yes these two are covered in the `systemd.unit` man page, https://www.freedesktop.org/software/systemd/man/latest/systemd.unit.html#StartLimitIntervalSec=interval and the `RestartSec` is covered in the `systemd.service` man page.. https://www.freedesktop.org/software/systemd/man/latest/systemd.service#RestartSec=

`StartLimitIntervalSec` sets the amount of time (in seconds) within which a service is allowed to restart, the `StartLimitBurst` controls how many re-starts are allowed within the time set by `StartLimitInterValSec` and the `RestartSec` sets the amount of time to wait before a restart, so if we set the `StartLimitIntervalSec` greater than `StartLimitBurst * RestartSec` then we'll never hit the restart limit which prevents further restarts

Reply:
no no, it'll retry after sleeping for `RestartSec` which is 10 seconds..